formbuilder/components/URLDisplay.js

```diff
@@ -6,12 +6,15 @@ export default function URLDisplay(props) {
   };
 
   const icon = props.type === "admin" ? "eye-close" : "bullhorn";
+  const label = props.type == "admin" ? "Admin link" : "Form link";
   const glyphicon = `glyphicon glyphicon-${icon}`;
 
   return (
-    <div className="input-group input-group-lg">
-      <span className="input-group-addon"><i className={glyphicon} /></span>
-      <input onClick={onClick} type="text" className="form-control" defaultValue={props.url} />
+    <div>
```

Comment:
Ah, I think I got it, you're reffering to the fact that labels for the links aren't vertically aligned, correct?